app/experimenter/nimbus-ui/src/components/ChangeApprovalOperations/mocks.tsx

```diff
@@ -36,7 +35,8 @@ export const BaseSubject = ({
       timeoutEvent,
       rejectChange,
       approveChange,
-      startRemoteSettingsApproval,
+      reviewUrl:
+        "http://localhost:8888/v1/admin/#/buckets/main-workspace/collections/nimbus-mobile-experiments/records",
```

Comment:
Nit: could try and use [`REVIEW_URL`](https://github.com/mozilla/experimenter/pull/5835/files#diff-430239984ea0d023eca01d487e1e14285726983f13689b031f324e529d398c27R17) here